enhancement(frontend): Update react-bootstrap & boostrap

thezane opened the pull request:
This patch updates react-bootstrap & bootstrap to versions 1.0.0-beta9 & 4.3.1 respectively.  Obsolete react-bootstrap components are replaced.

simpslandyy commented:
@jmzaleski why is git detecting that these two branches are different??

gabriellesc commented:
@simpslandyy It looks like github squashed these commits into a single commit that was merged into `develop`: https://github.com/uoft-tapp/tapp/commit/0e754b222a49a847fbfe931bba10b03673a5247d
However, the stats look right:
> Pull master into development branch (#154): 
... 
Showing 59 changed files with 16,886 additions and 18,166 deletions

> is everything really in develop?? #156 
... 
Files changed 59 
... 
+16,886 −18,166

jmzaleski commented:
I think we will have to close/toss this PR and somehow fastforward develop to where master is

Commit:
enhancement(frontend): Update react-bootstrap & boostrap

This patch updates react-bootstrap & bootstrap to versions 1.0.0-beta9 & 4.3.1 respectively.  Obsolete react-bootstrap components are replaced.
